helpers/deploymentConfig.ts

```diff
@@ -3701,7 +3701,14 @@ export const globalConfig: NetworkConfig = {
             vTokenReceiver: preconfiguredAddresses.arbitrumsepolia.VTreasury,
           },
         ],
-        rewards: [],
+        rewards: [
+          {
+            asset: "USDT",
+            markets: ["USDT"],
+            supplySpeeds: ["250000000"], // 250 USDT per day
+            borrowSpeeds: ["250000000"], // 250 USDT per day
```

Comment:
These params are per second on Arbitrum. With 86400 seconds per day, to distribute 250 USDT per day, I think these values should be `2893`

Reply:
Updated: https://github.com/VenusProtocol/isolated-pools/pull/396/commits/b6c0883d9db0866a72645ed5d31fdefe7cb20bd4